content/docs/setup/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Setup and Upgrade
+title: Setup
 description: How to deploy and upgrade Istio in various environments, for example Kubernetes and Consul.
```

Comment:
how about

"...environments such as Kubernetes and Consul".

Reply:
done